pkgs/applications/video/obs-studio/wlrobs.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, fetchhg, obs-studio, wayland }:
+
+stdenv.mkDerivation rec {
+  pname = "wlrobs";
+  version = "82e2b93c6f662dfd9d69f7826c0096bef585c3ae";
+  src = fetchhg {
+    url = "https://hg.sr.ht/~scoopta/wlrobs";
+    rev = "${version}";
```

Comment:
```suggestion
    rev = version;
```

Reply:
In order to comply with the unstable package version naming scheme, I changed the underlying code, and this code no longer exists.